Libs/MRML/Core/vtkMRMLProceduralColorNode.h

```diff
@@ -88,6 +88,12 @@ class VTK_MRML_EXPORT vtkMRMLProceduralColorNode : public vtkMRMLColorNode
   /// Only compares the color map (x->RGB mapping)
   static bool IsColorMapEqual(vtkColorTransferFunction* tf1, vtkColorTransferFunction* tf2);
 
+  ///
+  /// Reimplemented vtkMRMLColorNode::GetLookupTable() to convert
+  /// the transfer function to a look up table with 256 entries
```

Comment:
do not hardcode 256 (see above) - it's just as simple to define a member variable